src/components/inspectors/ElementDestination.vue

```diff
@@ -0,0 +1,204 @@
+<template>
+  <div>
+    <b-form-group :label="$t('Element Destination')">
+      <b-form-select id="" v-model="destinationType" @change="destinationTypeChange" data-test="element-destination-type">
+        <b-form-select-option :value="null" disabled>{{ $t('Select element destination') }}</b-form-select-option>
+        <option v-for="option in options" :key="option.value" :value="option.value">{{ $t(option.content) }}</option>
+      </b-form-select>
+      <small class="form-text text-muted">{{ $t("Enter the destination...") }}</small>
+    </b-form-group>
+
+    <form-multi-select
+      v-if="destinationType === 'customDashboard'"
+      :label="$t('Dashboard')"
+      name="Dashboard"
+      :helper="$t('Select the dashboard to show the summary of this request when it completes')"
+      v-model="customDashboad"
+      :placeholder="$t('Type here to search')"
+      :showLabels="false"
+      :allow-empty="false"
+      :options="dashboardList"
+      :loading="loading"
+      optionContent="title"
+      optionValue="url"
+      class="p-0 mb-2"
+      :validation="validation"
+      :searchable="true"
+      :internal-search="false"
+      :preserve-search="false"
+      :clear-on-select="false"
+      @search-change="searchChange"
+      data-test="dashboard"
+    />
+    <form-input
+      v-if="destinationType === 'externalURL'" 
+      :label="$t('URL')"
+      v-model="externalURL"
+      :error="getValidationErrorForURL(externalURL)"
+      data-cy="events-add-id"
+      :placeholder="urlPlaceholder"
+      :helper="$t('Determine de URL where the request will end')"
+      data-test="external-url"
+    />
+  </div>
+</template>
+
+<script>
+import debounce from 'lodash/debounce';
+export default {
+  props: {
+    options: {
+      type: Array,
+    },
+    value: {
+      type: String,
+      default: '',
+    },
+  },
+  name: 'ElementDestination',
+  
+  data() {
+    return {
+      loading: false ,
+      validation: '',
+      destinationType: null,
+      dashboards: [],
+      customDashboad: null,
+      
+      defaultValues: {
+        summaryScreen: null,
+        customDashboard: null,
+        processLaunchpad: `process-browser/${window.ProcessMaker.modeler.process.id}?categorySelected=-1`,
+        externalURL: null,
+        homepageDashboard: '/process-browser',
+        taskList: '/tasks',
+        taskSource: null,
+      },
+      urlModel: null,
+      local: null,
+      loadDashboardsDebounced: null,
+      urlPlaceholder: 'https://ci-ba427360a6.engk8s.processmaker.net/processes',
+      dashboardOptions: [
+        { value: 'first', title: 'Dashboard 1' },
+        { value: 'second', title: 'Dashboard 2' },
+      ],
+      externalURL: '',
+    };  
+  },
+  watch: {
+    customDashboad() {
+      this.setBpmnValues({
+        title: this.customDashboad.title,
+        url: this.customDashboad.url,
```

Comment:
Avoid using this in watch properties
```suggestion
    customDashboad(newValue) {
      this.setBpmnValues({
        title: newValue.title,
        url: newValue.url,
      });
    },
```

Reply:
fixed